tests/cli/drive-scanner.spec.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const m = require('mochainon');
+const drivelist = require('drivelist');
+
+describe('CLI: DriveScanner', function() {
+
+  describe('given no available drives', function() {
+
+    beforeEach(function() {
+      this.scanner = require('../../lib/cli/drive-scanner');
```

Comment:
Just realised... This can be a problem in code, not in tests. I mean, having a singleton with `start` and `stop` is not a good idea, because it will interact everywhere... Looking into it...